src/kata6.joinNames.js

```diff
@@ -1,3 +1,25 @@
-const joinNames = (namesObj) => {};
+const joinNames = (namesObj) => {
+let names = namesObj.map((x) => x.name);
+//.split(" "));
+let names1 = String(names);
+return names1.split(",").join(", ").replace(/,(?=[^,]+$)/, " &");
+};
+
+/* const joinNames = (namesObj) => {
+   return namesObj.map(object => object.name).join(", ").replace(/,(?=[^,]+$)/, " &");
+}; */
```

Comment:
Commented out code like this should never be left in, it just adds noise and doesnt add anything positive